downstream/modules/configuring_cost_mgmt-operator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// assembly_adding_ocp_sources.adoc
+[id="configuring_cost_mgmt-operator"]
+[[configuring_cost_mgmt-operator]]
+== Installing and configuring the Cost Management Operator
+
+The Cost Management Operator collects the data required for cost management.    
+
+Begin adding your OpenShift Container Platform cluster as a source to cost management, then install and configure the Cost Management Operator from OperatorHub.
+
+[NOTE]
+====
+See https://docs.openshift.com/container-platform/4.3/operators/olm-understanding-operatorhub.html[Understanding the OperatorHub] in the OpenShift documentation for more information about OperatorHub.
```

Comment:
```suggestion
See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/operators/olm-understanding-operatorhub[Understanding the OperatorHub] in the OpenShift documentation for more information about OperatorHub.
```

If you look at the "Develop" section you will find "Operators"
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/operators/index